app/Models/User.php

```diff
@@ -40,4 +51,51 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function isLibrarian()
+    {
+        return $this->role === self::LIBRARIAN_ROLE;
+    }
+
+    public function Role()
+    {
+        if($this->role === self::LIBRARIAN_ROLE)
+        {
+            return "Librarian";
+        }
+        elseif($this->role === self::GENERAL_USER_ROLE)
+        {
+            return "General User";
+        }
+    }
+    
+    public function isBanned()
```

Comment:
No need to have this method. Because you can simply check the `$this->is_banned` value to understand if the user is banned or not. `is_banned` property is self explanatory.